File: app/src/main/java/com/joeshuff/dddungeongenerator/generator/monsters/LegendaryAction.java
package com.joeshuff.dddungeongenerator.generator.monsters;

import com.google.gson.annotations.SerializedName;

public class LegendaryAction {
    String name;
    String desc;

    @SerializedName("attack_bonus")
    int attackBonus;

    public String getName() {
        return name;
    }

    public String getDesc() {
        return desc;
    }

    public int getAttackBonus() {
        return attackBonus;
    }
}

File: app/src/main/java/com/joeshuff/dddungeongenerator/generator/monsters/LegendaryAction.kt
package com.joeshuff.dddungeongenerator.generator.monsters

import com.google.gson.annotations.SerializedName

class LegendaryAction {
    var name: String? = null
    var desc: String? = null

    @SerializedName("attack_bonus")
    var attackBonus = 0

}